build/config.gni

```diff
@@ -68,7 +68,17 @@ if (is_win) {
       assert(brave_channel == "", "Unknown channel name")
     }
   } else {
-    _channel = "Development"
+    if (brave_channel == "nightly") {
+      # Temporary support PR's Static builds on nightly channel.
+      # CI expect the installer files (*Setup.exe) in the output dir are
+      # correspond passed brave_channel. Therefore to make Static+nightly work
+      # we should set _channel to "Nightly".
+      # TODO(atuchin): revert after PRs start to use the development channel.
+      _channel = "Nightly"
+      brave_app_guid = "{C6CB981E-DB30-4876-8639-109F8933582C}"
```

Comment:
can we either reorder this or add a var so we don't duplicate the app id?

Reply:
np, I'll replace.
IMHO it's ok to do such copy/paste for temporary code. Otherwise, we do some excessive blame changes for a useful code.

Reply:
Done